robber/matchers/called.py

```diff
@@ -0,0 +1,42 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class Called(Base):
+    """
+    expect(function).to.be.called()
+    """
+
+    def matches(self):
+        return Helper.is_mock(self.actual) and self.actual.called
+
+    def failure_message(self):
+        if not Helper.is_mock(self.actual):
+            return '{object} is not of type mock'.format(object=self.actual)
+        return 'Expected {function} to be called'.format(function=self.actual)
+
+
+class CalledOnce(Base):
+    """
+    expect(function).to.be.called_once()
+    """
+
+    def matches(self):
+        return Helper.is_mock(self.actual) and self.actual.call_count == 1
+
+    def failure_message(self):
+        if not Helper.is_mock(self.actual):
+            return '{object} is not of type mock'.format(object=self.actual)
+        return 'Expected {function} to be called once'.format(function=self.actual)
+
+
+class Helper:
+    @staticmethod
+    def is_mock(mock):
+        return type(mock).__name__ == 'Mock' or type(mock).__name__ == 'MagicMock'
```

Comment:
Two things of note here.

The code should be `type(mock) is Mock or type(Mock) is MagicMock`. Is there a reason to use the magic `__name__` directly?

Why didn't you use `isinstance` here? Checking an object with `type` explicitly means that any object inherited from `Mock` would fail the test. In our case, any child class of `Mock` is just as good as the parent class. In fact `MagicMock` is a child class of `Mock` and the code could simply be `isinstance(mock,Mock)` (a object of class `Foo` that is a child of `Bar` is also an instance of `Bar`).